README.adoc

```diff
@@ -0,0 +1,51 @@
+= veracode-gradle-plugin
+:toc:
+
+Veracode Gradle Plugin that automates Veracode application security scanning activities.
+
+As a set of Gradle tasks, it is meant to be usable either as a command line submission tool or integrated as part of a continuous integration build process.
+
+It helps perform the following tasks:
+
+* Perform Veracode submissions for applications.
+* Scan results for flaws.
+
+== Pre-Requisites
+
+* Veracode account & application to perform scanning.
+* Veracode Java API JAR file (copy to `buildSrc/lib` directory).
+The https://tools.veracode.com/integrations/API-Wrappers/Java/bin/VeracodeJavaAPI.zip[zip file] is found in the https://analysiscenter.veracode.com/auth/helpCenter/api/c_about_wrappers.html[documentation] from Veracode's website.
+* JDK 7+ is a requirement for the Veracode Java API.
+
+== Getting Started
+
+* Clone project.
+* Copy the `VeracodeJavaAPI.jar` (from Veracode) to a newly created `buildSrc/lib` directory.
+* Rename `sample-gradle.properties` to `gradle.properties` and edit its contents to setup Veracode credentials.
+* Execute `./gradlew tasks` to see available tasks.
+
+== LICENSE
```

Comment:
Lots of projects do it and since it is a public repo we might eventually include a CLA.
Also I believe the license should be part of every file in the repository so for the README I put it at the bottom. 

I'll merge for now, there are lots of README updates coming up anyway so we can defer a decision on it.